content/blog/pulumi-bloggers-oct-2019/index.md

```diff
@@ -0,0 +1,42 @@
+>
+>You’d never hire that way. So why, when choosing DevOps tools, do people find it acceptable to have different languages, DSLs that only key individuals understand?”
+
+Jon provides an in-depth review of Pulumi and includes the [code samples](https://github.com/hatech/demo-pulumi-python) he used on GitHub. Download the white paper from [HATech](https://hatech.io/pulumi-v-terraform-whitepaper-lp/).
+
+## VMWare VSphere and Pulumi
+
+![peek definition](peek-definition.jpg)
+
+[Cody De Arkland](https://twitter.com/Codydearkland) demonstrates how to create a VMWare VSphere Virtual Machine by adding a ResourcePool, DataStore, NetworkStore and the template with Pulumi. He also sets the configuration values such as DNS settings, default gateway, domain, and hostname all in code. All of this runs on the ESXi virtualization server and vCenter (which is why the Datacenter ID is needed for configuration). His example is available on [GitHub}(https://github.com/codyde/pulumi-vsphere-ts). We like Cody’s caveat that his post is not an endorsement but call to explore new tools. In his conclusion, he nails home the point why Infrastructure as Code is the future of DevOps.
+
+> “Infrastructure as Code tools like Pulumi (and others in the space, i.e. Terraform from HashiCorp) add a ton of value on top of traditional deployment methods. Being able to store your infrastructure deployments in source control (i.e. GitHub/GitLab) opens up the door to a number of collaboration concepts. When you partner this in with some form of a CI/CD tool (i.e. CircleCI) - things get really interesting. Being able to submit a PR to change an infrastructure deployment, and having something like CircleCI automatically deploy the resources when changes are merged creates a pretty powerful platform for delivering infrastructure to end-users.”
+
+## Pulumi Architecture and Examples
+
+For the francophones in our community, Thomas Schersach has written a three-part series that includes deploying Kubernetes and a serverless application. The [first post](https://blog.octo.com/decouvrir-les-cloud-native-languages-avec-pulumi/) is an overview of how Pulumi works as well as the role of Providers for resource deployments. The [second article](https://blog.octo.com/pulumi-par-la-pratique-kubernetes/) demonstrates how to deploy Amazon’s Elastic Kubernetes Services with Pulumi. In the [third installment](https://blog.octo.com/pulumi-par-la-pratique-serverless/), Thomas shows how to deploy a Lambda serverless function. Example code is available on [GitHub](https://github.com/Tirke/try-pulumi). Overall a great set of articles to get you up and running on Pulumi; and if you don’t speak or read French, Google Translate is your [friend](https://translate.google.com/translate?sl=fr&tl=en&u=https://blog.octo.com/author/thomas-schersach-thom/).
+
+## Bloggers Welcome
+
+If you write about your experience using Pulumi, let us know by tagging us on [Twitter](pulumicorp).
```

Comment:
This twitter link doesnt appear to work.

Reply:
link fixed